server/src/index.ts

```diff
@@ -76,6 +93,17 @@ const typeDefs = gql`
     type: DatabaseTypes!
   }
 
+  input AddEnvVarInput {
+    name: String!
```

Comment:
For both delete and add on the ui we will have acces to the appId only

Reply:
So flow is like this : 
- we pass appId, 
- we get app name in our db from the appId, 
- we pass it to dokku cli. 

Is this correct?

Reply:
Yup :+1: